web/src/pages/assets/index.tsx

```diff
@@ -1,16 +1,102 @@
+import { useEffect, useState } from "react";
+import { useAuth } from "@/hooks/useAuth";
+
 import Assets from "@/components/assets/Assets";
 import MetaHead from "@/components/meta/Head";
+import LoginFallback from "@/components/ui/LoginFallback";
+import Button from "@/components/ui/Button";
+import Loading from "@/components/ui/Loading";
+import Title from "@/components/ui/Title";
+
+import styles from "./index.module.css";
+
+export default function AssetsPage() {
+  const { authInProgress, isAuthed, userData } = useAuth();
+  const [studies, setStudies] = useState<Study[]>([]);
+  const [studiesLoading, setStudiesLoading] = useState(true);
+
+  const isApprovedResearcher = userData?.roles.includes("approved-researcher");
+
+  useEffect(() => {
+    if (!isApprovedResearcher) return;
+
+    const fetchStudies = async () => {
+      setStudiesLoading(true);
+      try {
+        // TODO: Replace with actual API call
+        // const response = await getStudies();
+        // setStudies(response.data);
+
+        // Dummy data for now
+        await new Promise((resolve) => setTimeout(resolve, 500)); // Simulate API call
+        setStudies([
+          {
+            id: "1",
+            title: "COVID-19 Research Study",
+            description: "A comprehensive study on COVID-19 impacts and treatments",
+          },
+          {
+            id: "2",
+            title: "Cancer Treatment Analysis",
+            description: "Analysis of various cancer treatment methodologies",
+          },
+        ]);
+      } catch (error) {
+        console.error("Failed to fetch studies:", error);
+        setStudies([]);
```

Comment:
This is just to give us something to work with for now. This is heavily dependant on https://github.com/ucl-arc-tre/portal/pull/152